src/main/java/duke/command/AddCommand.java

```diff
@@ -0,0 +1,36 @@
+package duke.command;
+
+import duke.DukeException;
+import duke.Storage;
+import duke.Ui;
+import duke.task.Task;
+import duke.task.TaskList;
+
+/**
+ * The AddCommand class represents a command
+ * that adds a new task to Duke's task list.
+ */
+public class AddCommand extends Command {
+    /** The task to be added to Duke */
+    private final Task task;
+
+    /**
+     * Constructs a new AddCommand.
+     *
+     * @param task The task to be added to Duke.
+     */
+    public AddCommand(Task task) {
+        this.task = task;
+    }
+
+    /**
+     * {@inheritDoc}
```

Comment:
Maybe you should add a few more comments on what is the different behavior of this overridden function? eg. saving to File etc. I noticed the same issue in several other places too.

Reply:
Admittedly got a bit lazy, will probably update the JavaDoc comments for the execute() methods in each subclass to better clarify what they do in a future commit :) Thanks!